media/css/devreg/reviewers.styl

```diff
@@ -1092,6 +1092,19 @@ table.activity .error {
     }
 }
 
+.review-actions-testedon {
+    .formset {
+        margin-top: 1.5em;
+    }
+    a.add-row {
+        background: url("../../img/icons/news-feed-sprite.png") no-repeat 0 -1444px;
```

Comment:
Wow, this is a huge sprite. We should try to get rid of it instead of relying on it more, or at least optimize its contents, we only need like 2 images from it, that's a bit ridiculous :)

Can you try to either a) use a separate image b) regenerate the `img/mkt/icons/mkt-reviewer-icons.png` to include what you need, and use that ? Bonus if you also fix `.log-attachments a` to stop using this sprite too